src/components/About/db.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "id": 1,
+    "title": "Orcasound App",
+    "path": "/images/about/iPhone+coffee - Copy.webp"
```

Comment:
Consider cleaning up the image filenames a bit (no need to have stuff like "copy")